TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -188,25 +186,40 @@ def execute_benchmark_workload(self, workload: str,
             client_options += (f",basic_auth_user:{self.auth_details['username']},"
                                f"basic_auth_password:{password_to_censor}")
         elif self.auth_type == AuthMethod.SIGV4:
-            raise NotImplementedError(f"Auth type {self.auth_type} is not currently support for executing "
-                                      f"benchmark workloads")
-        # Note -- we should censor the password when logging this command
-        # Fix commit used for OSB on latest verified working commit
-        workload_revision = "fc64258a9b2ed2451423d7758ca1c5880626c520"
+            raise NotImplementedError("SigV4 auth not supported for benchmark workloads")
+        shared_command = (f"--exclude-tasks=check-cluster-health "
+                          f"--target-host={self.endpoint} "
+                          f"--pipeline=benchmark-only "
+                          "--test-mode --kill-running-processes "
+                          f"--client-options={client_options}")
+        numeric_version = f"{self.version.split('_')[1]}.0"
+        major_version = int(numeric_version.split('.')[0])
+
+        flavor = self.version.split('_')[0]
+
+        # Use esrally for older ES versions, opensearch-benchmark for everything else
+        if flavor == "ES" and major_version in [1, 2, 5]:
```

Comment:
Before I tweaked the command to target ES5 or lower, I confirmed the command behavior